project-koku/koku
# [COST-4013] add order by account for AWS+OCPonAWS endpoints

## Jira Ticket

[COST-4013](https://issues.redhat.com/browse/COST-4013)

## Description

This change will add account to the order_by options for AWS endpoints

## Testing

1. Checkout Branch
2. Restart Koku
3. load customer data
4. hit http://localhost:8000/api/cost-management/v1/reports/aws/costs/?group_by[account]=*&order_by[account]=desc or http://localhost:8000/api/cost-management/v1/reports/aws/costs/?group_by[account]=*&order_by[account]=asc or http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/aws/costs/?group_by[account]=*&order_by[account]=desc or http://localhost:8000/api/cost-management/v1/reports/openshift/infrastructures/aws/costs/?group_by[account]=*&order_by[account]=desc
5. Should order appropriately 

## Notes
Before this change
{
    "order_by": {
        "account": [
            "Unsupported parameter or invalid value"
        ]
    }
}
...
